app/views/results/edit.html.erb

```diff
@@ -44,7 +44,7 @@
       this.style.display = 'none';
       window.onbeforeunload = null;
     });
-
+    
```

Comment:
@GitGeminiano Remember to review your PR diffs before submitting them. This file should not have been changed.